src/alphabets/mod.rs

```diff
@@ -374,6 +440,16 @@ where
     }
 }
 
+/// Returns the english ascii lower case alphabet.
+pub fn english_ascii_lower_alphabet() -> Alphabet {
+    Alphabet::new(&b"abcdefghijklmnopqrstuvwxyz"[..])
+}
+
+/// Returns the english ascii upper case alphabet.
+pub fn english_ascii_upper_alphabet() -> Alphabet {
+    Alphabet::new(&b"ABCDEFGHIJKLMNOPQRSTUVWXYZ"[..])
+}
+
```

Comment:
What's the use case for these two?

Reply:
Some of the available alphabets in this library are `atcgATCG` or the like. For my use case, I'd like to integer encode the sequences based on the alphabet. Then use that downstream in something like an embedding. So if the alphabet has lower and upper case, but the underlying sequences are only upper cases, it doubles the dimensionality. So having these plus the set operations make it pretty straightforward to strip out unneeded letters.